### PR TITLE
🪗  A new beautiful accordion

### PR DESCRIPTION
**Pull request by bri-meow**
Hello I would like to add this mui accordion, please ignore yarn lock it is auto generated, i added new dependencies.

Here it is open
Screen Shot 2022-01-29 at 11 06 52 AM

Here it is closed
Screen Shot 2022-01-29 at 11 05 33 AM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
```
Comment (bri-meow) [inline package.json:7]: do we really need this? i guess it's ok to have since the mui docs say to install it.

File: src/App.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import FirstTest from './Components/FirstTest';
 import CustomAccordion from './Components/CustomAccordion';
 import ReactStrapAccordion from './Components/ReactstrapAccordion';
 import ReactBootstrapAccordion from './Components/ReactBootstrapAccordion';
+import MUIAccordion from './Components/MUIAccordion';
```
Comment (bri-meow) [inline src/App.js:7]: i think this should be called `CatAccordion`

Comment (Sheilabob) [reply src/App.js:7]: No way.  I vote for KittenAccordion!

**Review by Sheilabob — APPROVED** [review]
Awesome - merge now!

**Pull request merged**